department-of-veterans-affairs/va.gov-cms
# VACMS-19463: Install, enable, and configure ga4 module.

## Description

Relates to #19463 

## Testing done
Local implementation and enabling of module.

## QA steps

What needs to be checked to prove this works?
Navigate to /admin/config/services/ga4-google-analytics and make sure the measurements_id is filled out and /help and /help/* are in the pages
What needs to be checked to prove it didn't break any related things?
Site should see no changed behavior.
What variations of circumstances (users, actions, values) need to be checked?

### Definition of Done


### Select Team for PR review



### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
